components/elm/cime_config/config_compsets.xml

```diff
@@ -133,6 +133,25 @@
     <alias>I1850CNPRDCTCBCTOP</alias>
     <lname>1850_DATM%QIA_ELM%CNPRDCTCBCTOP_SICE_SOCN_MOSART_SGLC_SWAV</lname>
   </compset>
+  <!---I E3SMv3 forcing for v3 land spinup compsets-->
+  <compset>
+    <alias>I1850E3SMv3CNTOP</alias>
+    <lname>1850_DATM%E3SMv3_ELM%CNRDCTCBCTOP_SICE_SOCN_MOSART_SGLC_SWAV</lname>
+  </compset>
+
+  <compset>
+    <alias>I1850E3SMv3CNPTOP</alias>
+    <lname>1850_DATM%E3SMv3_ELM%CNPRDCTCBCTOP_SICE_SOCN_MOSART_SGLC_SWAV</lname>
```

Comment:
What is "E3SMv3" supposed to convey in this compset?   We know its E3SM and "v3" is, again, not something we want in compsets in general.

Reply:
@rljacob  E3SMv3 meaning that the climate forcing files used to drive offline land simulation was from E3SMv3 coupled simulation coupler output, which we used  to drive the land BGC SPINUP simulation for v3 coupled simulations. we also added the E3SMv2 coupler output files as the land BGC v2 simulation forcing files (DATM%E3SMv2) in our BGCv2 compset branch, which I will submit all commits from the branch to master later. If I  change the compset short name such as I1850E3SMv3CNPTOP to I1850E3CPLCNPTOP, what is a good to distinguish the coupler output forcing from v2 and v3? Thanks.

Reply:
I see its forcing instead of an internal configuration.  But isn 't that what the CPLHIST option is for?

Reply:
I too believe this new compset is just what CPLHIST is for.